sdk/azidentity/errors.go

```diff
@@ -75,6 +75,23 @@ func (e *AuthenticationFailedError) Error() string {
 		fmt.Fprint(msg, "Response contained no body")
 	}
 	fmt.Fprintln(msg, "\n--------------------------------------------------------------------------------")
+	// adding link to TROUBLESHOOTING.md for relevant errors
+	var anchor string
+	switch e.credType {
+	case "AzureCLICredential":
+		anchor = "azure-cli"
+	case "ClientCertificateCredential":
```

Comment:
These values are unlikely to change but it's still helpful to use the constants so readers don't have to think about typos.

Reply:
Thank you! I didn't notice that detail. Fixed now.